templates/orderPickupCancelled.ejs

```diff
@@ -0,0 +1,152 @@
+
+                <!-- START CENTERED WHITE CONTAINER -->
+                <table class="main" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; background: #ffffff; border-radius: 3px;">
+
+                    <!-- START MAIN CONTENT AREA -->
+                    <tr>
+                        <td class="wrapper" style="font-family: sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;">
+                            <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+                                <tr>
+                                    <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Hi <%= firstName %>,</p>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">The pickup event for your order on the ACM membership portal has been cancelled:</p>
+                                            <p>Pickup event:</p>
+                                                <p><%= order.pickupEvent.title %></p>
+                                                <p><small>From <%= order.pickupEvent.start %> to <%= order.pickupEvent.end %></small></p>
+                                                <p><small><%= order.pickupEvent.description %></small></p>
+                                            <p>Order:</p>
+                                                <% for(var i=0; i < order.items.length; i++) { %>
+                                                                    <p><%= order.items[i].itemName %></p>
+                                                                    <p><small><%= order.items[i].salePrice %> x <%= order.items[i].quantityRequested %> = <%= order.items[i].total %></small></p>
+                                                <% } %>
+                                                <h1><%= order.totalCost %></h1>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Please reschedule a new pickup time for your order. If you are not able to schedule a pickup time
+                                          before the end of the quarter, then your order will be cancelled and fully refunded to your account.
+
+                                          You can reschedule here: https://
+                                        </p>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Regards, <br />ACM UCSD</p>
```

Comment:
@steets250 let me know what the URL for rescheduling an order should be on the frontend.

Reply:
https://members.acmucsd.com/store/orders?uuid=${order.uuid}